http-client-core/src/main/java/io/micronaut/http/client/ServiceHttpClientConfiguration.java

```diff
@@ -294,6 +317,13 @@ public static class ServiceConnectionPoolConfiguration extends ConnectionPoolCon
     public static class ServiceWebSocketCompressionConfiguration extends WebSocketCompressionConfiguration {
     }
 
+    /**
+     * The service HTTP/2 configuration.
+     */
+    @ConfigurationProperties(WebSocketCompressionConfiguration.PREFIX)
+    public static class ServiceHttp2Configuration extends Http2Configuration {
```

Comment:
does this need to be `@BootstrapCompatible`?

Reply:
I don't know. The other subconfigs arent either